go/vt/topo/tablet_test.go

```diff
@@ -34,43 +34,107 @@ import (
 // GetTabletsByCell first tries to get all the tablets using List.
 // If the response is too large, we will get an error, and fall back to one tablet at a time.
 func TestServerGetTabletsByCell(t *testing.T) {
+	const cell = "zone1"
+	const keyspace = "keyspace"
+	const shard = "shard"
+
 	tests := []struct {
-		name      string
-		tablets   int
-		opt       *topo.GetTabletsByCellOptions
-		listError error
+		name               string
+		createShardTablets int
+		expectedTablets    int
+		expectedShards     []string
+		opt                *topo.GetTabletsByCellOptions
+		listError          error
+		keyspaceShards     map[string][]string
 	}{
 		{
-			name:    "negative concurrency",
-			tablets: 1,
+			name: "negative concurrency",
+			keyspaceShards: map[string][]string{
+				keyspace: []string{shard},
```

Comment:
All of these `[]string` specifications are redundant given the defined type for the values in the map. Doesn't hurt anything though either, but it makes things a little tidier:
```
keyspace: {shard},
```

Reply:
I think this is why gofmt is failing:
```
Run $(go env GOPATH)/bin/golangci-lint run go/... --timeout 10m || exit 1
  $(go env GOPATH)/bin/golangci-lint run go/... --timeout 10m || exit 1
  shell: /usr/bin/bash -e {0}
go/vt/topo/tablet_test.go:53: File is not `gofmt`-ed with `-s` (gofmt)
				keyspace: []string{shard},
```

Reply:
Good idea, updated